incubator/group/types.proto

```diff
@@ -304,3 +304,82 @@ message GenesisState {
     option (gogoproto.goproto_stringer) = false;
     Params Params = 1 [(gogoproto.nullable)=false];
 }
+
+//
+// Queries
+//
+service Query {
```

Comment:
Do we want to be explicit with all query operations? We are not with commands (Messages). Commands are routed through handlers. We could also have query routes as in the sdk now or  #50